subsys/disk/Kconfig

```diff
@@ -144,6 +144,7 @@ config DISK_ACCESS_STM32_SDMMC
 	bool "STM32 SDMMC driver"
 	depends on HAS_STM32CUBE
 	select USE_STM32_HAL_SD
+	select USE_STM32_HAL_SD_EX
```

Comment:
if it's L4 specific, 
```
	select USE_STM32_HAL_SD_EX if SOC_SERIES_STM32L4X
```

Reply:
It's l4+ specific and not l4 specific. I think that it is better to select it globally as it is harmless

Reply:
@shlomow it's not harmless as `USE_STM32_HAL_SD_EX` doesn't exist on all series